Products/ZenModel/IpInterface.py

```diff
@@ -581,6 +581,14 @@ def getRRDTemplates(self):
         return []
 
 
+    def monitored(self):
+        '''
+        Return True if this instance should be monitored. False
+        otherwise.
+        '''
+        return False if self.adminStatus != 1 else self.monitor
```

Comment:
You should call the super method in the else clause incase the base class implementation ever changes.